db/schema.rb

```diff
@@ -56,10 +56,9 @@
     t.boolean "suboptimal", default: false, null: false
     t.bigint "primer_panel_id"
     t.integer "request_id"
-    t.integer "tag_depth", default: 1
     t.index ["library_id"], name: "index_aliquots_on_library_id"
     t.index ["primer_panel_id"], name: "index_aliquots_on_primer_panel_id"
-    t.index ["receptacle_id", "tag_id", "tag2_id", "tag_depth"], name: "aliquot_tag_tag2_and_tag_depth_are_unique_within_receptacle", unique: true
+    t.index ["receptacle_id", "tag_id", "tag2_id"], name: "aliquot_tags_and_tag2s_are_unique_within_receptacle", unique: true
```

Comment:
Do you need to remove tag depth? Does leaving it in have any side effects?

Reply:
The index name was hardcoded inside SS code originally, and that's why it has been changed in some files.
Tag_depth is removed in the schema but it has never been applied in production; the migration that added this column has been removed. We have to add again that migration when rolling back.